cmd/get_logs.go

```diff
@@ -362,6 +417,16 @@ func (glc *getLogsCmd) getCloudName() string {
 	return ""
 }
 
+func updateLinuxNodesList(nodeList []v1.Node, agentPoolName string) []v1.Node {
```

Comment:
this name can be more specific, something like `filterNodesFromPool` or similar may explain better what the func does.

Reply:
Updated